src/lib/action/entry-transform.ts

```diff
@@ -1,31 +1,52 @@
 import { APIAction } from './action'
 import { OfflineAPI } from '../offline-api'
 import Entry from '../entities/entry'
+import * as _ from 'lodash'
 
 class EntryTransformAction extends APIAction {
   private contentTypeId: string
   private fromFields: string[]
-  private toFields: string[]
-  private transform: Function
+  // private toFields: string[]
+  private transformEntryForLocale: Function
+  private shouldPublish: boolean
 
-  constructor (contentTypeId: string, fromFields: string[], toFields: string[], transform: Function) {
+  // constructor (contentTypeId: string, fromFields: string[], toFields: string[], transformation: Function) {
```

Comment:
Is this the "next generation constructor" or can we get rid of this?

Reply:
I think that for now we can just get rid of it